src/conn/conn_api.c

```diff
@@ -2647,6 +2647,7 @@ __conn_config_file_system(WT_SESSION_IMPL *session, const char *cfg[])
             WT_RET_MSG(
               session, EINVAL, "Live restore is not compatible with an in-memory connections");
 #ifdef _MSC_VER
+        /* FIXME-WT-14051 Add support for Windows */
         WT_RET_MSG(session, EINVAL, "Live restore is not supported on Windows");
```

Comment:
Live restore is now supported no?

Reply:
It probably would work, but I haven't removed the "not supported" WT_ERR_MSG call when we initialise the file system, or updated the section where we temporarily use the posix system during set up.

Given windows support is low priority and technically out of scope for the project I've limited this PR to only compiling under windows so we can get rid of the ugly #ifdefs

Reply:
Ah I see, understood!